editor/src/clj/editor/scene_visibility.clj

```diff
@@ -241,8 +241,11 @@
   (run [scene-visibility] (hide-outline-name-paths! scene-visibility (g/node-value scene-visibility :unselected-hideable-outline-name-paths))))
 
 (handler/defhandler :hide-toggle-selected :workbench
-  (active? [scene-visibility evaluation-context] true)
-  (enabled? [scene-visibility evaluation-context] true)
+  (active? [scene-visibility evaluation-context]
+           (g/node-value scene-visibility :active-scene-resource-node evaluation-context))
+  (enabled? [scene-visibility evaluation-context]
+            (or (g/node-value scene-visibility :selected-hideable-outline-name-paths evaluation-context)
+                (g/node-value scene-visibility :selected-showable-outline-name-paths evaluation-context)))
```

Comment:
The command shouldn't be enabled unless something is selected, as invoking it will cause an error. We also hide the command unless we're in the Scene View.